### ISSUE TITLE
Big blurry border around windows

### ISSUE THREAD

**Issue by Guolfo** [Dorako/pf2e-dorako-ui#255]
The border around the windows that are open on top of the map have a yellow blurry glow that is really big and has no transparency effect, making the UI appear cluttered when there is more than one window open. 
Some windows do not have this effect.

The issue can be seen on the Foundry VTT app

![Screenshot 2023-10-17 alle 18 43 40](https://github.com/Dorako/pf2e-dorako-ui/assets/148260737/ce54e0c1-1031-471f-902a-423cb844812f)

**Comment by Dorako** [comment]
I've seen one person with this issue, and they fixed it by:
> Ok, tested in a fresh world and cleared browser cache and did a hard reload.  The glow appears to be working correctly now.
There's supposed to be a glow, but it's minimal like this:
![image](https://github.com/Dorako/pf2e-dorako-ui/assets/1561190/a63e07f1-3554-4270-b3ab-b4001c025667)

**Comment by Guolfo** [comment]
I have the issue on the app, not on browser. 
I can't start the campaign from scratch and I tested on new one and still seeing the weird glow.
If it isn't fixable by other means, I'll use the old skin.

**Comment by Dorako** [comment]
The app is just a browser in disguise, so it's probably dependant on browser feature set.
You can toss this into the custom css section to use a normal shadow:
`.foundry2-pc.window-app,
.foundry2.window-app {
    box-shadow: 0px 1px 20px 3px black !important;
}`

**Comment by Guolfo** [comment]
This worked, thank you!

**Comment by Dorako** [comment]
The .foundry2-pc class has been replaced with .foundry2-restricted, making the css:
```css
.foundry2-restricted.window-app, .foundry2.window-app { box-shadow: 0px 1px 20px 3px black !important; }
```

**Comment by Guolfo** [comment]
Hi! With the new update I still got the issue. Is there a new css to fix it? Thank you!

**Comment by Dorako** [comment]
> Hi! With the new update I still got the issue. Is there a new css to fix it? Thank you!

Did you try the one from the above comment?

**Comment by Guolfo** [comment]
> > Hi! With the new update I still got the issue. Is there a new css to fix it? Thank you!
> 
> Did you try the one from the above comment?

Sure, it worked since last update.

**Comment by Dorako** [comment]
> > > Hi! With the new update I still got the issue. Is there a new css to fix it? Thank you!
> > 
> > 
> > Did you try the one from the above comment?
> 
> Sure, it worked since last update.

I'm confused. Do you mean 'until last update'?

**Comment by Guolfo** [comment]
> > > > Hi! With the new update I still got the issue. Is there a new css to fix it? Thank you!
> > > 
> > > 
> > > Did you try the one from the above comment?
> > 
> > 
> > Sure, it worked since last update.
> 
> I'm confused. Do you mean 'until last update'?

Sorry, my bad. English is not my first language and I made a mistake. 
the css isn’t currently working, and it stopped working just right after the latest update.
I now have the big blurry borders around the windows as seen on the original post.

**Comment by L4D15** [comment]
I have the same problem with the latest version, using the Foundry App.

**Comment by Dorako** [comment]
Can you reproduce in 3.0.2?

**Comment by L4D15** [comment]
Updating to 3.0.2 has fixed the issue. Now I see the glow correctly using the Foundry electron app.

**Comment by Dorako** [comment]
it might still be there for player sheets; I'll fix that if that's the case.

**Comment by L4D15** [comment]
Character sheets and NPCs still show the incorrect glow. I will try to test every possible window I can think of and report back.

**Comment by Dorako** [comment]
> Character sheets and NPCs still show the incorrect glow. I will try to test every possible window I can think of and report back.

Nah it's fine, I know exactly what is up. I'll fix.

**Comment by Dorako** [comment]
3.0.3 should have fixed it for all windows.

**Comment by L4D15** [comment]
Updated to 3.0.3 but characters sheets and NPCs still show the incorrect glow.
<img width="1624" alt="Captura de pantalla 2023-11-18 a las 21 58 48" src="https://github.com/Dorako/pf2e-dorako-ui/assets/2948600/1c91d44c-45b0-4478-8803-2fd7de8d3dee">

**Comment by Dorako** [comment]
Aww alright, guess I'll have to test with the electron client myself

**Comment by L4D15** [comment]
Let me know if I can help you in any way, I will keep an eye on the comments 👍

**Comment by Dorako** [comment]
Oh lol, silly mistake. I had fixed the code, but I didn't have sass running to produce the output css, so 3.0.2 did contain the right fix, it just wasn't actually effective.
3.0.4 contains the compiled css.

**Comment by L4D15** [comment]
I can confirm it's fixed on 3.0.4 👍
